### ISSUE TITLE
[turborepo] ERROR Unable to run generate - missing requirements (npx)

### ISSUE THREAD

**Issue by jpgilchrist** [vercel/turbo#5075]
### What version of Turborepo are you using?

1.9.8

### What package manager are you using / does the bug impact?

pnpm

### What operating system are you using?

Windows

### Describe the Bug

```
$ npm install turbo@latest -g
$ npx create-turbo@latest
$ cd {new-repo-path}
$ turbo gen workspace
```

outputs

```
ERROR Unable to run generate - missing requirements (npx)
```

### Expected Behavior

to create a new workspace and/or prompt with options

https://turbo.build/repo/docs/core-concepts/monorepos/code-generation

### To Reproduce

see description

**Comment by kashan-ahmad** [comment]
Facing the same issue regarding both a custom generator as well as with workspaces.

**Comment by tknickman** [comment]
Hey thanks for the report, can you try this with 1.10, and also, let me know the result of `which npx`?

Also, @kashan-ahmad are you on Windows as well?

Thank you!

**Comment by business-door** [comment]
Yep, I'm on windows and I got turbo@1.9.7, seems like windows is gonna cost me a few bucks since I miss bash already.

**Comment by ngopas** [comment]
same issue here

turbo version  - 1.10.1
npx version  9.6.4

**Comment by tknickman** [comment]
@ngopas, are you using windows as well?

**Comment by sofshant** [comment]
I'm using windows and also running into this issue

**Comment by tknickman** [comment]
Got it, I can reproduce on windows. Thanks for the report! 

I'll get a fix up asap

**Comment by tknickman** [comment]
This will release in canary shortly and we'll get a patch out hopefully tomorrow!

**Comment by tknickman** [comment]
Should be fixed in `1.10.2`

**Comment by sofshant** [comment]
Can confirm its fixed thanks!
